### PR TITLE
@orta => Adds tests for Stripe.

### PR DESCRIPTION
**Pull request by ashfurrow**
Try not to merge it before I'm read this time :stuck_out_tongue: 

Adds tests for #438.

### CODE REVIEW

**Comment by ashfurrow** [comment]
It's really just `ManualCreditCardInputViewModelTests.swift` for now.

**Comment by ashfurrow** [comment]
This is good for review. Sorry I didn't get to this sooner – this trip has been weird.

**Comment by ashfurrow** [comment]
Adds 54 tests, which is not bad. I'm happy.

File: KioskTests/App/RAC/RACFunctionsTests.swift [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            it("requires tld") {
+                let valid = stringIsEmailAddress("ash@artsymail") as Bool
+                expect(valid) == false
+            }
+
+            it("validates good emails") {
+                let valid = stringIsEmailAddress("ash@artsymail.com") as Bool
+                expect(valid) == true
+            }
+        }
```
Comment (orta) [inline KioskTests/App/RAC/RACFunctionsTests.swift]: you may find shared assertions in quick can reduce the LOC for these

https://github.com/Quick/Quick/blob/master/Documentation/SharedExamples.md#reducing-test-boilerplate-with-shared-assertions

**Comment by orta** [comment]
Looks good

**Pull request merged**